src/schema/paddle.ts

```diff
@@ -32,7 +32,7 @@ export const resolvers: IResolvers<unknown, BaseContext> = traceResolvers<
   Query: {
     pricingIds: (): PricingId => {
       return {
-        value: JSON.stringify(planTypes()),
+        value: JSON.stringify(remoteConfig.vars?.pricingIds),
```

Comment:
since you defined `value` as required do you want to do `|| {}` here so its always string object?

Reply:
`JSON.stringify(undefined)` is `undefined`